september-util/src/main/java/edu/asu/diging/gilesecosystem/septemberutil/service/ISystemMessageHandler.java

```diff
@@ -2,7 +2,7 @@
 
 public interface ISystemMessageHandler {
 
-    public abstract void handleError(String msg, Exception exception);
+    public abstract void handleException(String msg, Exception exception, String messageType);
```

Comment:
let's call that "handleMessage", given that there doesn't have to be an exception.